apps/vs-code-designer/src/app/commands/initProjectForVSCode/InitVSCodeStepBase.ts

```diff
@@ -81,7 +80,7 @@ export abstract class InitVSCodeStepBase extends AzureWizardExecuteStep<IProject
 
     context.telemetry.properties.isProjectInSubDir = String(isSubpath(context.workspacePath, context.projectPath));
 
-    const vscodePath: string = path.join(context.workspacePath, vscodeFolderName);
+    const vscodePath: string = path.join(context.workspacePath, '.vscode');
```

Comment:
I feel like you still need to update your branch with the latest changes of main. This was a change  I made 10 days ago and it seems like it is getting reverted here.